vumi/middleware/tests/test_message_storing.py

```diff
@@ -15,7 +15,8 @@ def setUp(self):
 
         # Create and stash a riak manager to clean up afterwards, because we
         # don't get access to the one inside the middleware.
-        self.persistence_helper.get_riak_manager()
+        manager = self.persistence_helper.get_riak_manager()
```

Comment:
But we're immediately calling ``add_cleanup(manager.close)`` here so we're not checking anyway? Not entirely sure I understood your response to my comment.